languages/ruby/exercises/concept/floating-point-numbers/.docs/hints.md

```diff
@@ -0,0 +1,14 @@
+### General
+
+### 1. Calculate the interest rate
+
+- Using an `if` or `case` statement can be useful when checking conditions.
```

Comment:
Will students have learnt about the `case` statement in an earlier exercise?

Reply:
Good point. I suspect we'll reference `case` in the after.md of conditionals, rather than have a seperate exercise.

But this PR should probably add that bit to the conditionals exercise to ensure to it (sorry to keep adding things @dvik1950!)

Reply:
That makes sense. I'm not able to find the `conditionals` exercise, or anything related to it though. I'm happy to fork/write one if no one else has done it or assigned it to themselves. I'll apply the other suggested changes later today :)

Reply:
Ah - yes. We don't have that yet! Good point :D

That's a good next one to fork then ;)

Reply:
:D